### ISSUE TITLE
New package homepage?

### ISSUE THREAD

**Issue by juanparadox** [juanparadox/react-tachyons#42]
Should we begin creating a new home page for this project in the same fashion that the devs who worked on the material react components and bootstrap react components did? We can either recreate the tachyons site using our components or create something new from the ground up.

Thoughts?

**Comment by anater** [comment]
Yeah probably. Like a gallery? Or like fully documented pages?

**Comment by juanparadox** [comment]
I guess it can be the fully documented pages? That way we have the components showing how tachyons has it, but we can also add our own 'how to' for each component if need be and we can include the proptypes and what not. What do you think?

**Comment by anater** [comment]
True. I like that approach. We should come up with a general outline for the different sections so we can start creating the documentation.

**Comment by juanparadox** [comment]
Take a look at the template branch locally that I've been working on and see if you like the approach I'm taking for showing the components (just a start). Basically there is now a home page and when the user clicks on components they will see a simple list where they can click on a specific component to see what it looks like. We can also just copy the tachyons site verbatim or if you have a design you'd like to show then we can explore that too.

**Comment by anater** [comment]
I like it a lot. Yeah, I think a big long page with **every** component is too much. It also seems like it'd take a lot more effort to break them into their own pages - not as familiar with react-router (is it hard to have a lot of pages?). Maybe a page per major component group and that way the components page is essentially a directory page (without images). That way we can keep using the {ComponentGroup}Example.jsx components for each page.

**Comment by juanparadox** [comment]
Having lots of pages isn't tough with react-router... I'll update the sample homepage with what you mentioned above and we can go from there.

**Comment by juanparadox** [comment]
What do you think so far? Anything we should change?
